doc/cutting-a-release.md

```diff
@@ -91,27 +91,14 @@ git push --set-upstream origin "${RELEASE}.x"
 
 **NOTE:** No code review or Pull Request is needed as part of this step.
 
-## Update the documentation links
-
-Pushing the branch should start the CI builds. You can find and watch the
-triggered builds at https://travis-ci.com/googleapis/google-cloud-cpp. One of
-the CI builds automatically pushes a new version of the documentation to the
-`gh-pages` branch. Wait for this build to complete, then you should checkout
-`gh-pages` and change the `index.html` page to link to this new version and
-commit it:
-
-```bash
-git checkout gh-pages
-git pull
-```
-
-Change the value of the meta tag's `URL=` parameter to refer to the
-`index.html` file in the directory matching the new release version number.
-
-```
-git commit -am "Update documentation to ${RELEASE}"
-git push
-```
+## Generate and upload the documentation to googleapis.dev
+
+Manually run a kokoro job
+`prod:cloud-devrel/client-libraries/cpp/google-cloud-cpp/refdocs` with
+specifying the branch name (e.g. `v0.11.x`) as `Committish` field. This job will
+generate and upload the doxygen documentation to the staging bucket for
+googleapis.dev hosting. The uploaded documentation will be live generally in an
```

Comment:
s/will be live generally/will generally be live/ ?

Reply:
Done